### ISSUE TITLE
Show download counts along with referrer

### ISSUE THREAD

**Issue by achapin** [itchio/itch.io#57]
In the analytics page, the number of visits per referrer is shown. It would be really helpful to see how many of those visits convert into a sale/download. That way, we can see which referrers are not just good for traffic, but also for conversions!

**Comment by aaronfranke** [comment]
Good idea. UI-wise, this could easily be added as a 2nd column next to Visits in the referrer section.

It may also be useful to have a purchases column, for games that have both demos and for-sale games on one page. This would allow you to know "try'd vs buy'd" if you get what I mean.
